interfaces/seccomp/backend.go

```diff
@@ -384,7 +384,7 @@ func generateContent(opts interfaces.ConfinementOptions, snippetForTag string, a
 }
 
 // NewSpecification returns an empty seccomp specification.
-func (b *Backend) NewSpecification(appSet *interfaces.SnapAppSet) interfaces.Specification {
+func (b *Backend) NewSpecification(appSet *interfaces.SnapAppSet, opts interfaces.ConfinementOptions) interfaces.Specification {
```

Comment:
Nitpick: This is named inconsistently in the code. I like "opts" more than "confinement," which is present in a few places above in this diff. Would you mind updating it all to "opts"?